apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -721,3 +721,20 @@ restart_http_listener(Scheme, AppName) ->
 
 http_mod_name(emqx_management) -> emqx_mgmt_http;
 http_mod_name(Name) -> Name.
+
+for_node(Fun, Node) ->
+    try list_to_existing_atom(Node) of
+        NodeAtom ->
+            Fun(NodeAtom)
```

Comment:
Q: should we catch potential errors from `Fun` in this `try` as well?

Reply:
no, this time, it only catches the bad atom exception.